certbot_dns_multi/_internal/dns_multi.py

```diff
@@ -42,14 +42,21 @@ def add_parser_arguments(
     ) -> None:
         super().add_parser_arguments(add, default_propagation_seconds)
         add("credentials", help="MultiDNS credentials INI file.")
+        add("nameservers", help="Recursive nameservers to use for DNS queries.")
 
     def more_info(self) -> str:
         return (
             "This plugin performs DNS challenges using one of the DNS providers "
             "available via lego"
         )
 
-    def _setup_credentials(self) -> None:
```

Comment:
I've had a chance to try this PR out, but there are some issues (like this line) which prevent it from working:

```
Can't instantiate abstract class Authenticator without an implementation for abstract method '_setup_credentials'
```

If you're still interested in working on it, please make sure that you've tested it before submitting it for review, and I'll give it another round.